components/Snackbar/src/private/MDCSnackbarMessageViewInternal.h

```diff
@@ -62,7 +63,8 @@ typedef void (^MDCSnackbarMessageDismissHandler)(BOOL userInitiated,
  that it needs to be dismissed prior to its timer-based dismissal time.
  */
 - (_Nonnull instancetype)initWithMessage:(MDCSnackbarMessage *_Nullable)message
-                          dismissHandler:(MDCSnackbarMessageDismissHandler _Nullable)handler;
+                          dismissHandler:(MDCSnackbarMessageDismissHandler _Nullable)handler
+                         snackbarManager:(MDCSnackbarManager *__weak)manager;
```

Comment:
I'm assuming you may have meant to add a nullability annotation here instead of `weak`?

Reply:
Nope, that's the autocomplete from Xcode. I prefer leaving it to Apple to determine the correct way to annotate method definitions based on their declaration.

Reply:
I believe Xcode auto-completed this because of the header API (that I'm assuming you wrote?). As best I can tell the `__weak` is a new addition to this code and is not something we typically annotate in our APIs anywhere.

Reply:
Holy cats this comment thread got all jumbled. My reply was meant to reply to the other thread also haha.

But cool cool!

Reply:
I'm surprised that we don't annotate weakly-retained parameters as such. I would think we like to be explicit about when we're not retaining something passed to an initializer.